### ISSUE TITLE
Gather api requirements (user stories) from diaspora and friendica developers

### ISSUE THREAD

**Issue by elf-pavlik** [w3c-social/social-interest#6]
moved from: https://www.w3.org/Social/InterestGroup/track/actions/1
@annando @jaywink my apologies on loosing track on this one :disappointed:  I also still need to catch up on your emails to social-web-comments list! Let's use this issue to make sure that our Use Cases include requirements from Friendica and Diaspora :exclamation:  @cwebber

**Comment by jaywink** [comment]
Wasn't this process done already, I thought user stories had been collected and voted on?

**Comment by elf-pavlik** [comment]
@jaywink yes, but in context of IG we can collect **all** stories and requirements that existing platforms already implemented :exclamation: 
- https://www.w3.org/wiki/Socialwg/Social_API/User_stories/Implementations

ATM we have no clue which features Diaspora and Friendica already implement and which you plan to add... Maybe I should also rename this issue to something clearer.
